cowprotocol/cowswap
# feat(swap): improve read only message for safe

Fixes #4284 

# Summary

Improves read-only mode message so users have a bit more clear what it is and what to do.

Before:
![image (34)](https://github.com/cowprotocol/cowswap/assets/2352112/afda22f1-82b3-49d6-9b3b-f749f35d99fe)


After:
![image (37)](https://github.com/cowprotocol/cowswap/assets/2352112/29dfb2d3-384c-4e0e-9f89-6209f4211e30)

![image (36)](https://github.com/cowprotocol/cowswap/assets/2352112/32fdd137-434e-4606-a5f0-8c531fe43da7)

# To Test

1. add this PR URL as safe app
2. without connecting a wallet in the safe, go to the app
3. check the read only message
